WORKSPACE

```diff
@@ -489,7 +489,7 @@ filegroup(
 )
     """,
     commit = ISTIO_API,
-    remote = "https://github.com/istio/api.git",
+    remote = "https://github.com/objectiser/istio-api.git",
```

Comment:
are the 2 changes in this file intended?

Reply:
This is temporary until the associated PR https://github.com/istio/api/pull/186 has been merged.